### PR TITLE
build: Upgrade to Vite 5

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/cli/src/decorators/Authorized.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { CONTROLLER_AUTH_ROLES } from './constants';
 import type { AuthRoleMetadata } from './types';
 
 export function Authorized(authRole: AuthRoleMetadata[string] = 'any'): Function {
-	return function (target: Function | Object, handlerName?: string) {
+	return function (target: Function | object, handlerName?: string) {
```
Comment (mutdmour) [inline packages/cli/src/decorators/Authorized.ts:6]: hmm is this expected?

Comment (cstuncsik) [reply packages/cli/src/decorators/Authorized.ts:6]: If the linter says. Apart from that it also looks correct to me. At least when I use objects as a type I use the lower case version like this.

**Review by mutdmour — APPROVED** [review]

File: packages/cli/src/decorators/Authorized.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/decorators/Authorized.ts:6]: `object` is correct here, no idea why I wrote `Object`. Thanks for fixing this.

**Comment by netroy** [comment]
Looks like this broke editor tests on node 20

**Comment by cstuncsik** [comment]
@netroy  I haven't seen it. Which one?

**Comment by netroy** [comment]
all builds on master.
[This](https://github.com/n8n-io/n8n/actions/runs/6983418478/job/19005311193) is the last one that failed.

**Comment by cstuncsik** [comment]
@netroy I this [PR](https://github.com/n8n-io/n8n/pull/7818) fixes it. Can you have a look?

**Comment by cstuncsik** [comment]
Tried with node@20 and worked. There must be some diff in the env.

**Comment by netroy** [comment]
now node 18 is failing as well
https://github.com/n8n-io/n8n/actions/runs/7005383973/job/19055090458

**Pull request merged**